velox/experimental/wave/exec/tests/CMakeLists.txt

```diff
@@ -59,46 +59,48 @@ target_link_libraries(
 
 add_test(velox_wave_exec_test velox_wave_exec_test)
 
-add_executable(velox_wave_benchmark WaveBenchmark.cpp)
+if(${VELOX_ENABLE_BENCHMARKS} OR ${VELOX_ENABLE_BENCHMARKS_BASIC})
```

Comment:
nit: I think `VELOX_ENABLE_BENCHMARKS` is sufficient here.

Reply:
I suppose it isn't a basic benchmark and should only be enabled along the top level benchmarks - the dependency is available then as well. But perhaps we can get a comment about it.